hw/ip/kmac/data/kmac.hjson

```diff
@@ -1010,9 +1010,11 @@
         byte-write: "true"
         desc: '''Message FIFO.
 
-              Any write to this window will be appended to the FIFO. Only lower
-              2 bits `[1:0]` of the address matter to writes within the window
-              in order to handle with sub-word writes.
+              Any write operation to this window will be appended to MSG_FIFO. SW can
```

Comment:
I always recommend to switch to 1 sentence per line whenever we change hjson and markdown files as this will simplify future diffs. The 100 or whatever chars per line is something we primarily did early in the project.

Reply:
I was abiding by that rule for doc changes, but I was not aware it also applied to hjson files. I will keep that in mind for future commits.